winml/test/api/LearningModelBindingAPITest.cpp

```diff
@@ -367,60 +367,60 @@ static void VerifyInvalidBindExceptions() {
   auto ensureWinmlSizeMismatch = std::bind(matchException, std::placeholders::_1, WINML_ERR_SIZE_MISMATCH);
   auto ensureWinmlInvalidBinding = std::bind(matchException, std::placeholders::_1, WINML_ERR_INVALID_BINDING);
 
-    /*
+  /*
         Verify tensor bindings throw correct bind exceptions
     */
```

Comment:
[nit] Is the opening/closing assignment deliberate?

```
  /*
        Verify tensor bindings throw correct bind exceptions
    */
```

--> ?

```
  /*
        Verify tensor bindings throw correct bind exceptions
  */
```

Reply:
Thanks. I will create a separate manual adjustment for this. I don’t think it’s intentional but the formatter wasn’t smart enough to fix it.

Reply:
https://github.com/microsoft/onnxruntime/pull/17163